reference/muscle.md

```diff
@@ -0,0 +1,48 @@
+## Muscle
+
+```
+Muscle {
+  field SFDouble maxRadius   0.2
+  field SFVec3f  startOffset 0 0 0
+  field SFVec3f  endOffset   0 0 0
+  field MFColor  colors      []            # idle (0), contracting (1), and relaxing(2) state colors
+  field SFBool   castShadows TRUE
+  field SFBool   visible     TRUE
+}
+```
+
+### Description
+
+A [Muscle](#muscle) node can be used to graphically display the contraction of an artificial muscle implemented using [Joint](joint.md) and [Motor](motor.md) nodes.
+The artificial muscle is represented using a spheroid where the symmetry axis is the vector between the joint's closest upper [Transform](transform.md) node (or `startOffset` position) and the `endPoint` [Solid](solid.md) node (or `endOffset` position).
```

Comment:
I would remove the "(or `start/endOffset` position)" as it looks confusing and anyhow it is better explained below.